keps/sig-node/2238-liveness-probe-grace-period/kep.yaml

```diff
@@ -0,0 +1,32 @@
+title: Liveness Probe Grace Period
+kep-number: 2238
+authors:
+  - "@ehashman"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: provisional
```

Comment:
Let me know if I should update this to implementable @derekwaynecarr or if we want a second PR to give folks another chance to TAL before KEP freeze

Reply:
I bumped it to implementable given how close we're getting to KEP freeze.